app/controllers/amend/StartAmendJourneyController.scala

```diff
@@ -56,6 +56,9 @@ class StartAmendJourneyController @Inject()(
             logger.error(exception.getMessage, exception)
             throw exception
         }
-      }).flatten
+      }).flatten.recover {
```

Comment:
If they have an enrolment, but it responds with an error on a get registration, we should show tech difficulties, not the NotRegistered.

The not registered should be used whenever they are in amend mode, on any page and don't have the enrolment. Connector shouldn't come into it